openedx/adg/lms/applications/tests/test_views.py

```diff
@@ -0,0 +1,446 @@
+"""
+Tests for all the views in applications app.
+"""
+
+from datetime import date
+
+import mock
+import pytest
+from django.test import RequestFactory, Client
+from django.urls import reverse
+
+from openedx.adg.lms.applications.models import ApplicationHub
+from openedx.adg.lms.applications.views import ApplicationHubView, ApplicationSuccessView
+from student.tests.factories import UserFactory
+from .constants import USERNAME, EMAIL, PASSWORD
+
+
+@pytest.mark.django_db
+@pytest.fixture()
+def user_for_testing(db):
+    """
+    Create a test user and their corresponding ApplicationHub object
+
+    Returns:
+        User object
+
+    """
+    user = UserFactory(username=USERNAME, email=EMAIL, password=PASSWORD)
```

Comment:
Same goes here, If we don't need specific name just use UserFactory().
When we are writing tests we don't add Returns or Arguments in docstrings.